raiden/accounts.py

```diff
@@ -10,10 +10,12 @@
 def find_datadir():
     home = os.path.expanduser("~")
     datadir = None
-    if sys.platform.startswith('linux'):
+    if sys.platform.startswith('linux') or sys.platform.startswith('FreeBSD'):
```

Comment:
Really os.name == "posix" is the way to check for a unix like operating system. See my original comment.